CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/InteractiveGLVisualProcessor.java

```diff
@@ -175,7 +167,7 @@ public VisualOperation setHitTestingEnabled(boolean enabled) {
     @Override
     public VisualOperation hitTestCursor(final int x, final int y, final HitState hitState, final Queue<HitState> notificationQueue) {
         hitTester.queueRequest(new HitTestRequest(x, y, hitState, notificationQueue, resultState -> {
-            if (resultState.getCurrentHitType().equals(HitType.NO_ELEMENT) && !DrawModeCursorFlag.DrawModeEnabled.get()) {
+            if (resultState.getCurrentHitType().equals(HitType.NO_ELEMENT) && !DrawModeCursorFlag.getDrawModeEnabled()) {
```

Comment:
This line can be replaced with the following:
```
            if (resultState.getCurrentHitType().equals(HitType.NO_ELEMENT) && !VisualGraphUtilities.isDrawingMode(GraphManager.getDefault().getActiveGraph())) {
```            
And this is the only code change required for this ticket.
All other code changes can be removed.
There is no need for the DrawModeCursorFlag class to exist at all.

Note that all the documentation changes can be kept.